scripts/init

```diff
@@ -91,89 +76,43 @@ sync_node() {
 }
 
 configure_nginx() {
+  gomplate -f /opt/wallarm/wallarm-status.conf.tmpl > /etc/nginx/conf.d/wallarm-status.conf
```

Comment:
- Please handle the case when `nginx.conf` is already mounted inside the container (if so do not run gomplate)
- Please, check all ENV variables from official docs once again
https://docs.wallarm.com/installation/inline/compute-instances/docker/nginx-based/#run-the-container-passing-the-environment-variables